build.gradle

```diff
@@ -20,7 +20,7 @@ allprojects {
   group = "com.netflix.spinnaker.rosco"
 
   spinnaker {
-    dependenciesVersion = "0.40.0"
```

Comment:
Yes. Released it here:
https://github.com/spinnaker/spinnaker-dependencies/tree/v0.41.0

On Wed, Jun 22, 2016 at 12:08 PM Eric Wiseblatt <notifications@github.com>
wrote:

> In build.gradle
> <https://github.com/spinnaker/rosco/pull/109#discussion_r68083312>:
>
> > @@ -20,7 +20,7 @@ allprojects {
> >    group = "com.netflix.spinnaker.rosco"
> >
> >    spinnaker {
> > -    dependenciesVersion = "0.40.0"
>
> Does this depend on a new version of dependencies?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spinnaker/rosco/pull/109/files/8030403093b72fa7cf8b96fcbe3bc6f454d77ed1#r68083312>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIC_1p61rr4rBMuewp1nzVtvPqzIREuFks5qOV3pgaJpZM4I782X>
> .
>